python/torch_mlir/compiler_utils.py

```diff
@@ -91,6 +91,14 @@ def model_to_fxgraph(model, *model_args, dtype = None, **model_kwargs):
     if dtype is not None:
         model.to(dtype)
 
+    # Needed for models like bigbird-roberta-base that adjust their config during
+    # runtime bigbird-roberta-base, e.g. saying
```

Comment:
Should this have been?
```suggestion
    # runtime saying, e.g.
```